### PR TITLE
Move from Travis CI to CircleCI

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues: 

* [AGIOS-581](https://issues.jboss.org/browse/AGIOS-581) - Move aerogear-ios-push from Travis CI to CircleCI

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - set -o pipefail &&
+      xcodebuild
+        CODE_SIGNING_REQUIRED=NO
+        CODE_SIGN_IDENTITY=
+        PROVISIONING_PROFILE=
+        -sdk iphonesimulator
+        -destination "platform=iOS Simulator,name=${SIMULATOR},OS=${OS}"
+        -workspace $XCODE_WORKSPACE
+        -scheme $XCODE_SCHEME_TEST
+        clean build test |
```
Comment (jcesarmobile) [inline circle.yml]: This test scheme is not configured for running, remove the `clean build` part to only run the tests, or edit the scheme to allow it to run

Comment (danielpassos) [reply circle.yml]: Sorry! Copy and paste detected :bowtie:

**Comment by danielpassos** [comment]
@jcesarmobile Fixed!

**Comment by jcesarmobile** [comment]
Sorry, what I meant was to just remove `clean build` from the second command, not the whole command
We still want to run the tests

**Pull request merged**